provider/datagen/src/transform/cldr/decimal/mod.rs

```diff
@@ -184,6 +190,170 @@ impl TryFrom<NumbersWithNumsys<'_>> for DecimalSymbolsV1<'static> {
     }
 }
 
+#[cfg(feature = "experimental")]
```

Comment:
Suggestion: Refactor this code such that you don't need to sprinkle `cfg`s in the whole file. Suggestion:

- transform/cldr/decimal/symbols.rs contains `DataProvider<DecimalSymbolsV1Marker>` and its dependencies
- transform/cldr/decimal/compact.rs contains `DataProvider<ShortCompactDecimalFormatDataV1Marker>` and its dependencies, and the Long form and its dependencies

Still keep things like compact_decimal_pattern.rs abstracted out into their own files, too.